src/e2e-test/regression-test/config/testcases-cda.js

```diff
@@ -0,0 +1,10 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+const cases = [
+    { dataFile: 'sample.cda', templateFile: 'ccd.hbs' }
+];
```

Comment:
We should try to add the existing test cases to increase the coverage

Reply:
Maybe we can hold off until we figure out how the data files corresponds to the template files.